### ISSUE TITLE
Option to show stars as Bayer letters only?

### ISSUE THREAD

**Issue by Atque** [Stellarium/stellarium#1169]
I sometimes lack a feature to show star labels only as their Bayer designation (η Ari, υ Cep etc). So like a checkbox or something in the skylore tab, such as "Show stars' labels as Bayer designations", so that when checked, it displays the label of e.g Betelgeuse as α Ori.

I know there's a way to fix this, simply by renaming the file star_names.fab. Then, all stars are designated as their Bayer letter or the HIP number. But I want an option within the program to toggle this.

I follow a few blogs where the authors mention "asteroid X is close to star (Bayer letter)". With Stellarium, it's hard to follow what stars are mentioned because of the proper names of stars. Brining up the search window every time is tiresome.

Perhaps letting stars that have no Bayer designation display their Flamsteed designation, and those that don't have Flamsteed either, show their HIP number?

**Comment by gzotti** [comment]
Most users seem to prefer proper names over catalog numbers. But the current switch "use additional starnames" could probably be modified to "Star names: none/one/all". We have dozens of ideas around this topic, just no time ATM for a thorough solution.

**Comment by alex-w** [comment]
Please check version 0.20.2.18074

**Comment by axd1967** [comment]
> Most users seem to prefer proper names over catalog numbers. But the current switch "use additional starnames" could probably be modified to "Star names: none/one/all". We have dozens of ideas around this topic, just no time ATM for a thorough solution.

So this issue could also have been tagged as "Opiniuon"... :stuck_out_tongue: I also like 07f72860010dfa376e7afd64d9c039eff1f81ced
